.travis.yml

```diff
@@ -27,6 +27,10 @@ matrix:
       python: 3.7
     - env: TOXENV=py37-extra-deps
       python: 3.7
+    - env: TOXENV=py38
+      python: 3.8
+    - env: TOXENV=py38-extra-deps
```

Comment:
We agreed to keep one -extra-deps env, though we didn't decide which one, it doesn't matter that much.

Reply:
Oh, kmike told me to change that initially. I see now that he put a comment to the resolved conversation disregarding the suggestion after I committed it. I'll revert it back to using the 3.7 extra dependency environment.